client/src/utils.js

```diff
@@ -1,9 +1,12 @@
-import { tsvParse } from  "d3-dsv";
+import http from "./services/httpService";
```

Comment:
as per comment below, `http` shouldn't be imported here. It should be in the service file.

Reply:
Done